dotcom-rendering/src/components/MultiBylines.tsx

```diff
@@ -0,0 +1,94 @@
+import { css } from '@emotion/react';
+import type { ArticleFormat } from '../lib/articleFormat';
+import type { EditionId } from '../lib/edition';
+import type { ArticleElementRenderer } from '../lib/renderElement';
+import { palette } from '../palette';
+import type { ServerSideTests, Switches } from '../types/config';
+import type {
+	MultiByline as MultiBylineModel,
+	StarRating,
+} from '../types/content';
+import type { TagType } from '../types/tag';
+import { MultiByline as MultiBylineItemComponent } from './MultiByline';
```

Comment:
Should this be `MultiBylineComponent`? Or even just `MultiByline` since there's no name collision with the type, which has been renamed up on line 8?